store/auth.go

```diff
@@ -51,10 +51,32 @@ var (
 	UbuntuoneRefreshDischargeAPI = ubuntuoneAPIBase + "/tokens/refresh"
 )
 
+// a stringish is something that can be a []string or a string
+// like the values of the "extra" documents in error responses
+type stringish []string
```

Comment:
Can we name this as stringList instead?  stringish forces one to read the documentation at all times to have any idea of what it means (most things are "stringish" :).